coreos-4/policies/AU-Audit_and_Accountability/component.yaml

```diff
@@ -28,44 +28,26 @@
 - control_key: AU-2
   standard_key: NIST-800-53
   covered_by: []
-  implementation_status: planned
+  implementation_status: partial
   narrative:
     - key: a
       text: |
-        CoreOS was designed to satisfy the audit requirements outlined
-        in the Configuration Annex to the Protection Profile for General
-        Purpose Operating Systems available at https://www.niap-ccevs.org/Profile/PP.cfm.
+        A control response is planned. Engineering progress can be tracked via:
 
-        It is possible to extend CoreOS auditing beyond the events
-        outlined in the Configuration Annex. A control response
-        outlining the events CoreOS is capable of auditing is planned.
-        Progress can be tracked via:
-
-        https://github.com/ComplianceAsCode/redhat/issues/841
+        https://jira.coreos.com/browse/CMP-155
     - key: b
       text: |
-        Coordinating the security audit function with other
-        organizational entities is outside the scope of
-        CoreOS configuration.
```

Comment:
Many control responses, such as this one, seem valuable and descriptive. What's the reasoning to remove them?

Reply:
@shawndwells Added the control response back for controls deemed `not applicable`